README.md

```diff
@@ -1,69 +1,46 @@
 
-## Build and run in docker
+## S4S reference stack
+
+The test suite is one component of the [Sync for Science reference stack](https://github.com/sync-for-science/reference-stack-docker), which contains additional services such as DSTU2 and STU3 FHIR servers. The `SMART EHR` servers in the vendor list of the test suite refer to the servers that run with the reference stack, so they will be unavailable if you choose to run the test suite independently. *Additionally*, the FHIR resource validation steps will fail when the test suite is run independently, because those FHIR servers are used to perform the validation.
+
+## Build and run in Docker
+
+We use [Docker](https://www.docker.com/) to set up all the required services and run the test server.
 
     git clone https://github.com/sync-for-science/test-suite
     cd test-suite
     docker build -t tests .
     docker run --rm -it \
-      -e BASE_URL=http://tests.dev.syncfor.science:9003 \
-      -p 5000:5000 \
+      -p 9003:5000 \
       tests
-      
-### Develop in docker
 
-    docker run --rm -it \
-      -e BASE_URL=http://tests.dev.syncfor.science:9003 \
-      -p 5000:5000 \
-      -v /host/path/to/test-suite:/usr/src/app \
-      tests \
-      /bin/bash
+`9003` is the default port, but you can configure it to run on other ports - just be sure to add `-e BASE_URL=http://localhost:<port>` to the Docker command as well.
       
-... and from inside docker, run "behave" to execute tests.
+### Develop in Docker
 
-## Installation
+If you want to develop on the app using Docker, mount the repository on the container. Note that the container still has to be restarted to reflect any changes.
 
-This suite was developed for python 3. To install dependencies run `pip install -r requirements.txt`.
-
-To configure your own project, copy `behave.ini.dist` to `behave.ini`, and change the relevant configuration values.
+    docker run --rm -it \
+      -p 9003:5000 \
+      -v /host/path/to/test-suite:/usr/src/app \
+      tests
 
 ### Building the js app (for development)
 
-Install npm, then `npm install` and `npm run-script build`. If you're editing
+Install [npm](https://www.npmjs.com/), then `npm install` and `npm run-script build`. If you're editing
 JS, you can run `npm run-script watch` to automatically re-build when files
 change.
 
-## Running the test suite
+## Bloom filter
 
-To run the suite, run `behave` with `VENDOR` like `smart` or another value from https://github.com/sync-for-science/test-suite/tree/master/config.
+The test suite uses a [Bloom filter](https://en.wikipedia.org/wiki/Bloom_filter) to validate known codes. The filter is built with the `data/build_bf.py` script, but requires a number of external databases to be downloaded first. As a convenience, we maintain a prebuilt filter, which is downloaded every time the container starts.
```

Comment:
This is a functionality comment rather than a docs comment, but: it would be good to have the test suite look for the file at a known path and download only if absent. That allows people to mount the file from docker, or to build a new docker image that pre-bakes a version of the file in.